Library/Formula/nvm.rb

```diff
@@ -31,14 +31,19 @@ def caveats; <<-EOS.undent
     #{prefix} will destroy any nvm-installed Node installations
     upon upgrade/reinstall.
 
+    If upgrading from a previous version, please remember to activate the
+    new version by restarting your shell or by executing the command below:
+
+      . $(brew --prefix nvm)/nvm.sh
```

Comment:
It is. You'll need to activate the new nvm script in the shell environment after upgrading.